app/resources/api/v1/job_resource.rb

```diff
@@ -110,6 +112,10 @@ def work_plan_comment
       def barcode
         @model.container.barcode
       end
+
+      def set_uuid
+        @model.set&.uuid
```

Comment:
Shouldn't it be @model.set_uuid?, or do you need to build the instance of the set for any reason?

Reply:
You are right, I don't need an instance of the set. Thanks!